Makefile

```diff
@@ -131,14 +136,14 @@ build-aws-vpc-cni:    ## Build the VPC CNI container using the host's Go toolcha
 
 # Build VPC CNI plugin & agent container image.
 docker:	setup-ec2-sdk-override	   ## Build VPC CNI plugin & agent container image.
-	docker build $(DOCKER_BUILD_FLAGS) \
+	docker build $(call DOCKER_BUILD_FLAGS,primary) \
```

Comment:
personally i feel this is unnecessarily completed..imaging when we need provide more different DOCKER_BUILD_FLAGS for `cni-init `in the future..

I'd just define two variable, DOCKER_BUILD_FLAGS_CNI, DOCKER_BUILD_FLAGS_CNI_INIT and use them respectively.

Reply:
Per discussion, updated the DOCKER_BUILD_FLAGS.